### DISCUSSION TITLE
Textblock TextAlignment Issue

### DISCUSSION THREAD

**Discussion by cok666n** [AvaloniaUI/Avalonia#11405] [Q&A] [answered]
Hi, for starters I am a total newbie when it comes to Avalonia and/or WPF

Ive built this app that sometimes invokes an user control to display notifications. Here is the bulk of the XAML for this control :

`
<RelativePanel Background="{Binding BackgroundColor}" Opacity="{Binding IsVisible}">

		<RelativePanel.Transitions>
			<Transitions>
				<DoubleTransition Property="Opacity"  Duration="0:0:0.2"></DoubleTransition>
				<BrushTransition Property="Background"  Duration="0:0:0.2"></BrushTransition>
			</Transitions>
		</RelativePanel.Transitions>

		<TextBlock RelativePanel.AlignVerticalCenterWithPanel="True"
				   RelativePanel.AlignHorizontalCenterWithPanel="True"
				   FontSize="26"
				   FontWeight="Bold"
				   Opacity="0.9"
				   Text="{Binding NotificationMessage}"
				   VerticalAlignment="Center"
				   HorizontalAlignment="Center"
				   TextAlignment="Center"
				   ></TextBlock>
	</RelativePanel>
`

However sometimes the notification text (bind from the view model) has a \n and thus 2 lines.
I would like the text to be centered on both lines, but it seems whatever I do I always get the text on the left of the control. 

![image](https://github.com/AvaloniaUI/Avalonia/assets/3584469/7731f779-49ab-40c9-af76-32207345b096)

Im sure there is some WPF thing Im missing, but I felt like TextAlignment=Center should've done the trick.

Can anybody point me in the right direction?

**Comment by cok666n** [comment, marked as answer]
Oh well, it seems that going up from 11.0.0.preview 7 to preview 8 fixed that problem!

Side note, I can't seem to find a changelog for those previews, where would that be ?

**Reply by timunie** [reply]
There is no official one for preview builds. But you can check the diffs and PRs merged since preview7
